modules/c++/nitf/include/nitf/IOHandle.hpp

```diff
@@ -46,14 +46,6 @@ class IOHandle : public IOInterface
 
 public:
 
-    /*!
```

Comment:
It's been commented out since 2013 (https://github.com/mdaus/nitro/blob/b94f17d9670a5963d3a1f8d8be52b50e1eaa1ebe/c%2B%2B/nitf/include/nitf/IOHandle.hpp), so nothing should.